z2/src/validators.rs

```diff
@@ -46,31 +46,16 @@ impl Validator {
 }
 
 #[derive(Debug)]
-pub struct StakeDeposit {
-    validator: Validator,
-    amount: u8,
+pub struct ClientConfig {
```

Comment:
Ugly diff for simple change: created a `ClientConfig` and moved its members from  `StakeDeposit`